Documentation/operations/metrics.rst

```diff
@@ -115,6 +115,10 @@ section for the full list of available metrics and their options.
 The port of the Hubble metrics can be configured with the
 ``hubble.metrics.port`` Helm value.
 
+Note: Hubble emits http metrics only for pods that have the
```

Comment:
maybe like below (e.g. use Note section)

```suggestion
.. Note::

    Hubble emits http metrics only for pods that have the
    ``io.cilium.proxy-visibility`` annotation. Refer to
    :ref:`Layer 7 Protocol Visibility <proxy_visibility>` for details.
```

Reply:
I think using `.. Note::` as suggested by @sayboras is a good idea. 

I would phrase the sentence slightly differently however to avoid mentioning the specific label required as this is all explained with more details in the link.

What about something like this?

> L7 metrics, such as HTTP, are only emitted for pods which have layer 7 protocol visibility enabled. Refer to :ref:`Layer 7 Protocol Visibility <proxy_visibility>` for details.